assets/xml/objects/object_harfgibud.xml

```diff
@@ -24,19 +27,19 @@
         <DList Name="gHarfgibudGibdoLeftUpperArmDL" Offset="0x6EA0" />
         <DList Name="gHarfgibudGibdoAbdomenDL" Offset="0x6F98" />
 
-        <Texture Name="object_harfgibud_TLUT_0070A0" OutName="tlut_0070A0" Format="rgba16" Width="16" Height="16" Offset="0x70A0" />
-        <Texture Name="object_harfgibud_Tex_0072A0" OutName="tex_0072A0" Format="i8" Width="64" Height="32" Offset="0x72A0" />
-        <Texture Name="object_harfgibud_Tex_007AA0" OutName="tex_007AA0" Format="ci8" Width="8" Height="8" Offset="0x7AA0" />
-        <Texture Name="object_harfgibud_Tex_007AE0" OutName="tex_007AE0" Format="ci8" Width="8" Height="16" Offset="0x7AE0" />
-        <Texture Name="object_harfgibud_Tex_007B60" OutName="tex_007B60" Format="ci8" Width="8" Height="8" Offset="0x7B60" />
-        <Texture Name="object_harfgibud_Tex_007BA0" OutName="tex_007BA0" Format="ci8" Width="8" Height="8" Offset="0x7BA0" />
-        <Texture Name="object_harfgibud_Tex_007BE0" OutName="tex_007BE0" Format="ci8" Width="32" Height="32" Offset="0x7BE0" />
-        <Texture Name="object_harfgibud_Tex_007FE0" OutName="tex_007FE0" Format="ci8" Width="16" Height="16" Offset="0x7FE0" />
-        <Texture Name="object_harfgibud_Tex_0080E0" OutName="tex_0080E0" Format="rgba16" Width="16" Height="16" Offset="0x80E0" />
-        <Texture Name="object_harfgibud_Tex_0082E0" OutName="tex_0082E0" Format="i8" Width="8" Height="8" Offset="0x82E0" />
-        <Texture Name="object_harfgibud_Tex_008320" OutName="tex_008320" Format="rgba16" Width="8" Height="8" Offset="0x8320" />
-        <Texture Name="object_harfgibud_Tex_0083A0" OutName="tex_0083A0" Format="ci8" Width="16" Height="8" Offset="0x83A0" />
-        <Texture Name="object_harfgibud_Tex_008420" OutName="tex_008420" Format="i8" Width="8" Height="8" Offset="0x8420" />
+        <Texture Name="gHarfgibudTLUT" OutName="harfgibud_tlut" Format="rgba16" Width="16" Height="16" Offset="0x70A0" />
```

Comment:
With the `Gibdo` form assets I think it is a good idea to prefix it with `Gibdo` (like what I did for the skeleton) just to help signify which it is.